docs/add-ins/get-and-set-recurrence.md

```diff
@@ -0,0 +1,190 @@
+seriesTimeObject.setStartTime(10,30);
+seriesTimeObject.setDuration(30);
+
+var pattern = {
+    "seriesTime": seriesTimeObject,
+    "recurrenceType": "weekly",
+    "recurrenceProperties": {"interval": 1, "days": ["tue", "thu"], "firstDayOfWeek": "sun"},
+    "recurrenceTimeZone": {"name": "Pacific Standard Time"}};
+
+Office.context.mailbox.item.recurrence.setAsync(pattern, callback);
+
+function callback(asyncResult)
+{
+    console.log(JSON.stringify(asyncResult));
+}
+```
+
+> [!NOTE]
+> Only the appointment organizer can set the recurrence.
+
+## Get recurrence
+
+### Get recurrence as the organizer
+
+In the following example in compose mode, the appointment organizer can get the recurrence object of an appointment series given the series item or an instance of that series.
+
+```js
+Office.context.mailbox.item.recurrence.getAsync(callback);
+
+function callback(asyncResult){
+    var context = asyncResult.context;
+    var recurrence = asyncResult.value;
+
+    if (recurrence == null) {
+        console.log("Non-recurring meeting");
+        return;
+    }
+
+    console.log(JSON.stringify(recurrence));
+}
+```
+
+The following is an example of the results of the `getAsync` call on a series.
+
+> [!NOTE]
+> In this example, the `seriesTimeObject` is a placeholder for the JSON representing the `recurrence.seriesTime` property. You should use the [`SeriesTime`][SeriesTime link] methods to get the recurrence date and time properties.
+
+```js
+Recurrence = {
+    "recurrenceType": "weekly",
+    "recurrenceProperties": {
+        "interval": 1,
+        "days": ["tue","thu"],
+        "firstDayOfWeek": "sun"},
+    "seriesTime": {seriesTimeObject},
+    "recurrenceTimeZone": {
+        "name": "Pacific Standard Time",
+        "offset": -480}}
+```
+
+### Get recurrence as an attendee
+
+In the following example, an appointment attendee can get the recurrence object of an appointment series given the series, an instance of that series, or a meeting request.
+
+```js
+outputRecurrence(Office.context.mailbox.item);
+
+function outputRecurrence(item) {
+    var recurrence = item.recurrence;
+    var seriesId = item.seriesId;
+
+    if (recurrence == null) {
+        console.log("Non-recurring item");
+        return;
+    }
+
+    console.log(JSON.stringify(recurrence));
+}
+```
+
+The following is an example of the `item.recurrence` property.
+
+> [!NOTE]
+> In this example, the `seriesTimeObject` is a placeholder for the JSON representing the `recurrence.seriesTime` property. You should use the [`SeriesTime`][SeriesTime link] methods to get the recurrence date and time properties.
+
+```js
+Recurrence = {
+    "recurrenceType": "weekly",
+    "recurrenceProperties": {
+        "interval": 1,
+        "days": ["tue","thu"],
+        "firstDayOfWeek": "sun"},
+    "seriesTime": {seriesTimeObject},
+    "recurrenceTimeZone": {
+        "name": "Pacific Standard Time",
+        "offset": -480}}
+```
+
+### Access the recurrence details
```

Comment:
"Access" to me implies you can get or set the data. Is that the case here, or is this really "Get the recurrence details"?